### PR TITLE
Tooling

### PR DESCRIPTION
**Pull request by paultranvan**

### CODE REVIEW

File: tools/simulation/tracking.js [hunk 1]
```diff
@@ -184,3 +194,12 @@ export const createUser = async (serverURL, user) => {
...
 }
+
+module.exports = {
+  makeStartTransitions,
+  makeStopTransitions,
+  uploadToUsercache,
+  createUser,
+  translateToEMissionLocationPoint,
+  translateToEMissionMotionActivityPoint
+}
```
Comment (Crash--) [inline tools/simulation/tracking.js:205]: Why did you change from ESM to CommonJS? :(

File: tools/simulation/tracking.js [hunk 2]
```diff
@@ -64,14 +64,23 @@ export const makeStopTransitions = ts => {
...
   )
+  if (withForceStop) {
+    transitions.push(
+      transition(
+        'STATE_WAITING_FOR_TRIP_START',
+        'T_FORCE_STOP_TRACKING',
+        ts + 0.06
+      )
+    )
+  }
```
Comment (Crash--) [inline tools/simulation/tracking.js:75]: not really a chore here ^^

Comment (paultranvan) [reply tools/simulation/tracking.js:75]: Indeed!

File: tools/simulation/tracking.js (hunk 1 above)
Comment (paultranvan) [reply tools/simulation/tracking.js:205]: Because I had issues with eslint complaining about the module, and being already short on schedule, I didn't want to take time to investigate, so I took the easy path for this very small sub-project.
Hopefully I will have some time later to revert this and, more importantly, being able to import files directly from the submodule (I didn't succeed on this, hence this file which is a basically a reduced version of https://github.com/cozy/cozy-flagship-app/blob/master/src/app/domain/geolocation/tracking/tracking.js)
